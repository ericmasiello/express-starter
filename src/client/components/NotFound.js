/* @flow */
import React, { Component } from 'react';
import styles from './NotFound.scss';

// Note: Top level components must be class based for HMR to function
// eslint-disable-next-line react/prefer-stateless-function
export default class NotFound extends Component {
  render() {
    return (
      <div className={styles['not-found']}>
        <h1>404 - Page Not Found</h1>
      </div>
    );
  }
}
