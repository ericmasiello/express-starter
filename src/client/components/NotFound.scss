.not-found {
  padding: 0 1rem;
}
