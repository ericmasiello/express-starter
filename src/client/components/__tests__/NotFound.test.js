import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../NotFound';

it('should render', () => {
  const component = renderer.create(
    <NotFound />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
