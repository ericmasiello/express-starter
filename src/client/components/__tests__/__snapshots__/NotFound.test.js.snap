// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className={undefined}
>
  <h1>
    404 - Page Not Found
  </h1>
</div>
`;
